src/js/verifyUser.jsx

```diff
@@ -1,62 +1,52 @@
-import React from "react";
+import React, { Suspense, useEffect } from "react";
 import ReactDOM from "react-dom";
+import { useTranslation } from "react-i18next";
 import { ThemeProvider } from "@emotion/react";
 
 import AccountForm from "./components/AccountForm";
 
-import { getLanguage, jsonRequest } from "./utils";
+import i18n from "./i18n"; // to init localization...
+import { jsonRequest } from "./utils";
 import { THEME } from "./constants";
 
-class PasswordReset extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      localeText: {},
+function VerifyUser({ token, email }) {
+  const { t } = useTranslation();
+
+  useEffect(() => {
+    const verifyEmail = async () => {
+      const res = await jsonRequest("/verify-email", {
+        token,
+        email,
+      });
+      if (res === "") {
+        alert(t("users.verified"));
+        window.location = "/login";
+      } else {
+        console.error(res);
+        console.log("error, ", res);
+        console.log("translation, ", `users.${res}`);
+        alert(t(`users.${res}`) || t("users.errorCreating"));
+        window.location = "/password-forgot";
```

Comment:
There's no `password-forgot` page anymore. I think this should be `/password-request` or `/password-reset`. Come to think of it, what's the difference between those two pages?

Reply:
It looks like we link to `/password-request` from the Login page, but I don't see any places where we link to `/password-reset`

Reply:
The user start with requesting a new password in `/password-request`, a token get sent to his email to verify it, clicking it gets him to `/password-reset` where he puts the new password.

Reply:
Ah, I see. Thanks.